src/main/java/com/sequenceiq/cloudbreak/service/stack/resource/gcc/builders/GccResourceBuilderInit.java

```diff
@@ -69,14 +71,19 @@ public GccDeleteContextObject decommissionInit(Stack stack, Set<String> decommis
         GccCredential credential = (GccCredential) stack.getCredential();
         Compute compute = gccStackUtil.buildCompute(credential, stack);
         List<Resource> resourceList = new ArrayList<>();
+        List<InstanceGroup> instanceGroups = Lists.newArrayList(stack.getInstanceGroups());
         for (String res : decommissionSet) {
-            resourceList.add(new Resource(ResourceType.GCC_INSTANCE, res, stack));
+            resourceList.add(new Resource(ResourceType.GCC_INSTANCE, res, stack, instanceGroups.get(0).getGroupName()));
         }
         List<Resource> result = new ArrayList<>();
         for (Resource resource : resourceList) {
-            Instance instance = gccInstanceResourceBuilder.describe(stack, compute, resource);
-            for (AttachedDisk attachedDisk : instance.getDisks()) {
-                result.add(new Resource(ResourceType.GCC_ATTACHED_DISK, attachedDisk.getDeviceName(), stack));
+            try {
+                Instance instance = gccInstanceResourceBuilder.describe(stack, compute, resource, GccZone.valueOf(stack.getRegion()));
+                for (AttachedDisk attachedDisk : instance.getDisks()) {
+                    result.add(new Resource(ResourceType.GCC_ATTACHED_DISK, attachedDisk.getDeviceName(), stack, instanceGroups.get(0).getGroupName()));
+                }
+            } catch (IOException ex) {
+                LOGGER.error("There was a problem with the describe instance on Google cloud");
```

Comment:
shouldn't we do something with this exception? I don't think it will behave correctly if we leave it as it is - the result won't contain the attachedDisks.